test/checkTestReport.test.ts

```diff
@@ -0,0 +1,7 @@
+import { checkTestReport } from '../src/checkTestReport.ts';
+import { createTestReport } from '../src/createTestReport.ts';
+
+Deno.test('checkTestReport', async () => {
+  await createTestReport('./test_data/deno_success/testreport.json');
+  await checkTestReport('./test_data/deno_success/testReport.json');
```

Comment:
check that we have a match. Also check when we do not have a match (although thats already done in execute.test